dev_guide/managing_images.adoc

```diff
@@ -490,6 +490,13 @@ parameter is not specified, {product-title} sets it based on the image's tag:
 . Otherwise, {product-title} defaults `*imagePullPolicy*` to `*IfNotPresent*`.
 endif::[]
 
+[NOTE]
+====
+When using the `*Never*` Image Pull Policy, you can ensure that private images can only be used by pods with credentials to pull those images
+using the xref:../admin_guide/image_policy.adoc#image-policy-always-pull-images[*AlwaysPullImages* admission controller].
+If this admission controller is not enabled, any pod from any user on a node can use the image without any authorization check against the image.  
```

Comment:
s/pod/Pod/

Reply:
@bobfuru I am going to leave these as lower-case. The other references to _pod_ are all lower-case in this topic.